Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Drust Crisis

## Changelog:

- Added Drust culture in Vrykul group
- Added new crisis in Kul Tiras: Drust Incursion (Heartsbane part requires a sick married holder of d_drustwar, Gorak Tul can be summoned via decision once Heartsbane forms and wins the defensive holy war vs. Order of Embers or is an independent Throsic level 7 magic user that holds de jure d_drustwar)
- Added a new Tidemother holy order: Order of Embers (requires the Heartsbane Coven to form)
- Added a new Old Way holy order: The Thornspeakers (requires Gorak Tul to be summoned)
- Added two new bloodlines for Gorak Tul and Ulfar

## TODO:
- [x] Add decision icon for the summoning of Gorak Tul
- [x] Add missing localization
- [x] Event/manpower balancing tweaks
- [x] Localization proofreading
- [x] German localization (courtesy of Arithon)

# How to test:
- Play as Arthur Waycrest in all the startdates and write 2 commands: `add_trait cancer` and `event WCCRS.100` to test the event flow. Alternatively, the event itself should fire if Arthur Waycrest has an illness and Meredith is alive (or any other married counterparts that hold `d_drustwar`). Check whether the balance of the attackers and defenders is not overtuned/undertuned, but generally the event is supposed to be challenging for the defender, and provide enough manpower for the drust to win if facing a much more powerful enemy, since it can happen at any time. If possible, try to fire the event in a 'balkanized' Kul Tiras as well to check how it fares in such setting.
- Alternatively, conquer the entire de jure territory of Duchy of Drustvar as an independent Throsic follower (has to be level 7 magic user or higher) and attempt to summon Gorak Tul without the 'initial' event. To make your character Throsic use a console command, which is `religion throsic`.
- The above two should lock the crisis event from firing again. Please verify if that is the case.
- Verify if characters spawn correctly or are properly assigned to holy order titles.
- Also verify if localization is correct not only for Drust story events, but also Throsic religious events and general necromantic events, since these use dynamic localization.
